packages/modules/viewer-react/src/components/BaseViewer.tsx

```diff
@@ -41,24 +41,30 @@ export const BaseViewer: React.FC<ViewerProps> = ({
   additionalI18nNamespaces,
   additionalRpcInterfaces,
 }: ViewerProps) => {
-  const [authorized, setAuthorized] = useState(false);
+  // assume authorized when using a local snapshot
+  const [authorized, setAuthorized] = useState(snapshotPath ? true : false);
```

Comment:
```suggestion
  const [authorized, setAuthorized] = useState(!!snapshotPath);
```

Reply:
Not a fan of double negation (less readable) but if you're going to be picky about it :-)